index.html

```diff
@@ -1472,6 +1501,61 @@ <h2>
         feature/default allowlist=] is [=default allowlist/'self'=].
       </p>
     </section>
+      <section data-cite="html">
+        <h2>
+          Emulation
+        </h2>
+        <aside class="addition" id="a4">
+          <span class="marker">Candidate Addition:</span> Introduce emulated
+          position data to enable user-agent automation.
+        </aside><ins cite="#a4">
+        <p>
+          For the purposes of user-agent automation and application testing, this
+          document defines geolocation emulations.
+        </p>
+        <p>
+          Each [=top-level traversable=] has an associated
+          <code>"emulated position data"</code>, which is data representing
+          {{GeolocationCoordinates}} or null, initially null.
```

Comment:
This needs to be able to represent the state where the emulated position data means "no position available".

Reply:
done